src/service/action-impl.js

```diff
@@ -726,6 +748,23 @@ class ParserTokenizer {
       return {type: TokenType.LITERAL, value, index: newIndex};
     }
 
+    // Object literal.
+    if (c == OBJECT_SET[0]) { // '{'
+      let end = -1;
+      for (let i = newIndex + 1; i < this.str_.length; i++) {
```

Comment:
Yea, that's left to the `amp-bind` parser. I was conflicted between naming this `OBJECT_SET` and `EXPRESSION_SET`, but it *technically* is an object literal.